x-pack/solutions/security/plugins/security_solution/public/detections/components/attacks/filters/filters_section.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { type Dispatch, memo, type SetStateAction, useCallback, useMemo } from 'react';
+import type { Filter, TimeRange } from '@kbn/es-query';
+import { TableId } from '@kbn/securitysolution-data-table';
+import type { FilterGroupHandler } from '@kbn/alerts-ui-shared';
+import type { DataView, DataViewSpec } from '@kbn/data-views-plugin/common';
+import { PageFilters } from './page_filters';
+import type { AssigneesIdsSelection } from '../../../../common/components/assignees/types';
+import { useDataTableFilters } from '../../../../common/hooks/use_data_table_filters';
+import { useDeepEqualSelector } from '../../../../common/hooks/use_selector';
+import { useGlobalTime } from '../../../../common/containers/use_global_time';
+import { inputsSelectors } from '../../../../common/store/inputs';
+import {
+  buildAlertAssigneesFilter,
+  buildShowBuildingBlockFilter,
+  buildThreatMatchFilter,
+} from '../../alerts_table/default_config';
+import type { Status } from '../../../../../common/api/detection_engine';
+
+export interface FiltersSectionProps {
+  /**
+   * List of assignees retrieved from the assignees button on the alert page
+   */
+  assignees: AssigneesIdsSelection[];
```

Comment:
I think for this PR I would not add any code related to assignees. We do not have the ability to set assignees to attacks yet, this will be done via another PR (see [this ticket](https://github.com/elastic/kibana/issues/243155) and [that one](https://github.com/elastic/kibana/issues/243157)).

If I'm not mistaken, right now you're only using those to build filters (`buildAlertAssigneesFilter`) so it is safe to remove